app/jobs/import_versions_job.rb

```diff
@@ -102,6 +111,13 @@ def import_record(record)
     version.update_different_attribute(save: false)
     version.save
 
+    if version.uuid_previously_changed?
+      log(object: version, operation: :created, row: row)
+    else
+      log_version_operation = { 'merge' => 'merged', 'replace' => 'replaced' }.fetch(@import.update_behavior) { @import.update_behavior }
```

Comment:
Minor nit: can you pull the mapping object out into the class here so we don't allocate a whole bunch of these? Or just use `@import.update_behavior` directly (being able to map to a specific value for the logs is a nice touch, though).